src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx

```diff
@@ -502,6 +521,8 @@ export const BorrowInput = ({
           cryptoAmount={lendingQuoteData?.quoteBorrowedAmountCryptoPrecision ?? '0'}
           fiatAmount={lendingQuoteData?.quoteBorrowedAmountUserCurrency ?? '0'}
           isReadOnly
+          handleIsInputtingFiatSellAmountChange={handleToggleBorrowAssetIsFiat}
```

Comment:
for all of these, we could possibly just pass the toggle directly in, no?

```suggestion
          handleIsInputtingFiatSellAmountChange={toggleBorrowAssetIsFiat}
```

Reply:
Absolutely - was under the impression arity (1 vs. 0) would make TS yell at us but she's happy - https://github.com/shapeshift/web/pull/6469/commits/b7dc9b7923eb43e6e0f7b33063148c6e784a5b41